comm_norms.adoc

```diff
@@ -101,16 +121,19 @@ ____
 
 In this description, we immediately learn:
 
-* What Kubernetes is (a system for working with containerized applications, including deployment, scaling and management) 
-* How Kubernetes is abbreviated (little details like this one immediately deepen the comfort level of new arrivals; no one is expected to already know an arcane acronym upon arrival)
-* Where Kubernetes was developed (Google is noted as the originator of this code base, establishing the project as focused on enterprise applications and providing confidence that the software is well designed and maintained)
-* Kubernetes is open source (a user can expect to use, run, modify, and share changes to the code base; any questions about barriers to entry due to licensing fees, procurement processes, etc., are dismissed)
-* The project values community engagement (one can expect that contributions of code, documentation, etc.
-are welcome and encouraged)
+* What Kubernetes is. (It's a system for working with containerized applications, including deployment, scaling and management.) 
+* How Kubernetes is abbreviated. (Little details like this one immediately deepen the comfort level of new arrivals; no one is expected to already know an arcane acronym upon arrival.)
+* Where Kubernetes was developed. (Google is noted as the originator of this code base, establishing the project as focused on enterprise applications and providing confidence that the software is well designed and maintained.)
+* Kubernetes is open source. (A user can expect to use, run, modify, and share changes to the code base; any questions about barriers to entry due to licensing fees, procurement processes, etc., are dismissed.)
+* The project values community engagement. (One can expect that contributions of code, documentation, etc.
+are welcome and encouraged.)
 
 === Getting started
 
-The processes for creating good "getting started" documentation—occasionally called "onboarding documentation"—are outside the scope of this chapter (see the dedicated "Onboarding" section in this guidebook for more detail).
+The processes for creating good "getting started" documentation—occasionally called "onboarding documentation"—are outside the scope of this chapter.
+PD--A general observation here: There are several em dashes in this paragraph (and the chapter as a whole). I love em dashes. They are my favorite punctuation mark. However, when used too often, their effectiveness is diluted. Consider replacing them with other punctuation throughout the chapter.
```

Comment:
Tagging as a future to do for a later iteration of the chapter.

Reply:
I'm removing the comment inline and will start a separate style guide around this. We use emdashes quite a bit in other chapters, way way more than here, and that can be OK if we are consistent with how sparking/unsparingly we use them vs. commas, parenthesis, etc.

Instead we'll figure out how we want to use them editorially, then do some search function to find all them for manual adjustments if needed. I'm moving this action therefore out of this specific document and into the overall guide.